routes/payroll/controllers.js

```diff
@@ -0,0 +1,41 @@
+import PayrollService from "../../services/payroll.js";
+import httpResponse from "../../utils/httpResponse.js";
+
+const controller = {
+    add: async (req, res) => {
+        const addResponse = await PayrollService.add(req.body);
+        if (addResponse.message === "success") {
+            return httpResponse.CREATED(res, addResponse.data);
```

Comment:
From where you are getting statusCode?

Reply:
Sir the status code is in the httpResponse and i am just using it according to the need.These are the methods and i am calling the methods.